implementations/code_df/open_issue_age.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+
+import pandas as pd
+import numpy as np
+
+import utils
+from issue import Issue
+
+
+class OpenIssueAge(Issue):
+
+    def __init__(self, data_list, date_range=(None, None)):
+        """
+        Class for Age of Open Issues
+
+        :param data_list: A list of dictionaries, each element a line from the
+            JSON file
+        :param date_range: A tuple which represents the start and end date of
+            interest
+        """
+        super().__init__(data_list, date_range)
+
+        self.df = self._add_open_issue_age_col(self.df)
+
+    def compute(self):
+        """
+        Compute the average open issue age for all issues in the Perceval data.
+
+        :returns avg_open_issue_ages: the average age of open
```

Comment:
I think the variable name should be singluar `avg_open_issue_age` since it's a single value and not, say, a list

Reply:
OK. I was going for "average" of all of open issue ages. But yours makes more sense. Will change it.

Reply:
Done